.github/workflows/nightly.yml

```diff
@@ -0,0 +1,25 @@
+name: Nightly Release
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  delete_tag:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Delete release
+        uses: dev-drprasad/delete-tag-and-release@v0.2.1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          delete_release: true
+          tag_name: 1.0.0-SNAPSHOT
+  nightly:
+    needs: delete_tag
+    uses: Checkmarx/ast-jetbrains-plugin/.github/workflows/release.yml@main
+    with:
+      tag: "1.0.0-SNAPSHOT"
```

Comment:
Since we have a specific channel for nightly, SNAPSHOT might not be needed.
Does Jetbrains allow these suffixes in versions?
Even for testing purposes, let's name it 2.0.0-SNAPSHOT as our plugin versions are 2.0.X